keps/prod-readiness/sig-storage/1487.yaml

```diff
@@ -0,0 +1,5 @@
+kep-number: 1487
+beta:
```

Comment:
Let's remove beta - leave only stable, as we didn't do beta approval back then.
Stable is what you need now.

Reply:
Same for other PRs

Reply:
friendly ping